### DISCUSSION TITLE
JWT Token expiry configuration [Jun 10]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5921] [Discord Threads]
> 🙋‍♂️ **LK131**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Set it up as an environment variable during boot-up  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1105788027684536401)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
